packages/editor/src/components/post-actions/actions.js

```diff
@@ -59,11 +60,6 @@ function isTemplateRemovable( template ) {
 		! template.templatePart?.has_theme_file
 	);
 }
-const canDeleteOrReset = ( item ) => {
-	const isTemplatePart = item.type === TEMPLATE_PART_POST_TYPE;
-	const isUserPattern = item.type === PATTERN_TYPES.user;
-	return isUserPattern || ( isTemplatePart && item.isCustom );
-};
```

Comment:
I've removed this check.

## Why?

The "Reset" action is only available for the `wp_template` and `wp_template_part` entities.